src/System.Private.CoreLib/src/System/RtType.cs

```diff
@@ -1005,9 +1006,12 @@ private RuntimeType[] PopulateInterfaces(Filter filter)
                             if (constraint.IsInterface)
                                 al.Add(constraint);
 
-                            Type[] temp = constraint.GetInterfaces();
+                            Type?[] temp = constraint.GetInterfaces();
                             for (int j = 0; j < temp.Length; j++)
-                                al.Add(temp[j] as RuntimeType);
```

Comment:
Got compiler warning for type casting using "as" operator and updated them to use "is" instead